engine/class_modules/priest/sc_priest_discipline.cpp

```diff
@@ -264,10 +263,9 @@ void priest_t::create_buffs_discipline()
                                ->set_default_value( talents.sins_of_the_many->effectN( 1 ).percent() )
                                ->set_duration( talents.sins_of_the_many->duration() );
 
-  buffs.shadow_covenant = make_buff( this, "shadow_covenant", talents.shadow_covenant->effectN( 4 ).trigger() )
-                              ->set_trigger_spell( talents.shadow_covenant );
-
-  buffs.spirit_shell = make_buff( this, "spirit_shell", talents.spirit_shell );
+  buffs.shadow_covenant =
+      make_buff( this, "shadow_covenant", talents.discipline.shadow_covenant->effectN( 4 ).trigger() )
```

Comment:
did you check that the spell ID in effect 4 trigger of the talent is still correct?

Reply:
I believe it is:

```
#4 (id=810245)   : Trigger Spell (64): Shadow Covenant
                   Base Value: 0 | Scaled Value: 0 | PvP Coefficient: 1.00000 | Trigger Spell: 322105 | Target: Self (1)
```